java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -2393,6 +2561,48 @@ public void testSetInitialCapacity() {
     }
   }
 
+  @Test /* ViewVarCharVector */
+  public void testSetInitialCapacityInViews() {
```

Comment:
@lidavidm I have a few doubts about this test case, appreciate a closer look. 
The allocation behavior for views and non-views are a bit different since we have the 16bytes offset.

Reply:
I updated the test case to have a completely separate test criterion.